server.js

```diff
@@ -180,11 +180,12 @@ server.route([
       var polyfills = PolyfillSet.fromQueryParam(features || 'default', flags);
       var params = {
         features: polyfills.get(),
-        minify: true
+        minify: true,
+        unknown: 'polyfill'
```

Comment:
This changes the behaviour of the polyfill service to default to using all of the polyfills when it cannot detect the user-agent correctly.